packages/website/ts/containers/sol_compiler_documentation.ts

```diff
@@ -45,13 +43,12 @@ const docsInfoConfig: DocsInfoConfig = {
         [docSections.usage]: UsageMarkdown,
     },
     sectionNameToModulePath: {
-        [docSections.compiler]: ['"deployer/src/compiler"'],
-        [docSections.deployer]: ['"deployer/src/deployer"'],
-        [docSections.types]: ['"deployer/src/utils/types"', '"types/src/index"'],
+        [docSections.compiler]: ['"sol-compiler/src/compiler"'],
+        [docSections.types]: ['"sol-compiler/src/utils/types"', '"types/src/index"'],
     },
     menuSubsectionToVersionWhenIntroduced: {},
     sections: docSections,
-    visibleConstructors: [docSections.compiler, docSections.deployer],
+    visibleConstructors: [docSections.compiler],
```

Comment:
Below we have `Web3: constants.URL_WEB3_DOCS`. Is Web3 still part of the public interface of `sol-compiler`?

Reply:
What about `ContractInstance`?